pkgs/by-name/xa/xarcan/package.nix

```diff
@@ -85,6 +85,10 @@ stdenv.mkDerivation (finalPackages: {
     xtrans
   ];
 
+  patches = [
+    ./0001-fix-segfault.patch
```

Comment:
**Document why this patch is needed**

Reply:
Done

Reply:
Please use the upstream fix for that crash instead of your workaround: https://github.com/letoram/xarcan/commit/ecc4d0a6408dfeb19934e3bfd4c382b0862c03b4